package.json

```diff
@@ -42,7 +42,7 @@
   },
   "homepage": "https://github.com/fastify/fastify-nextjs#readme",
   "peerDependencies": {
-    "next": "12.x.x"
+    "next": "12.x.x || 13.x.x"
```

Comment:
should we support 12.x if we no longer are running tests on it? wonder if we should just do a semver bump

Reply:
I'd also be ok to run tests for both versions in CI. As an example, check out https://github.com/pinojs/hapi-pino/blob/fb705176199cef412de2634e5f554410a79cf852/.github/workflows/test.yml.

Reply:
Not sure how to do that with https://github.com/fastify/workflows/blob/main/.github/workflows/plugins-ci.yml?

Reply:
let's drop the shared workflow for this

Reply:
@kibertoad @mcollina updated with new job for next 12